src/renderer/src/routes/pages/Portfolio/index.tsx

```diff
@@ -1,27 +1,51 @@
 import { useTranslation } from 'react-i18next'
-import { TbRefresh } from 'react-icons/tb'
-import { BalanceChartPanel } from '@renderer/components/BalanceChartPanel'
-import { BlockchainDataPanel } from '@renderer/components/BlockchainDataPanel'
+import { MdAdd } from 'react-icons/md'
+import { TbFileExport, TbFileImport } from 'react-icons/tb'
+import { Outlet } from 'react-router-dom'
 import { IconButton } from '@renderer/components/IconButton'
-import { useAccountsSelector } from '@renderer/hooks/useAccountSelector'
-import { useBalancesAndExchange } from '@renderer/hooks/useBalancesAndExchange'
-import { useWalletsSelector } from '@renderer/hooks/useWalletSelector'
+import { Separator } from '@renderer/components/Separator'
+import { useModalNavigate } from '@renderer/hooks/useModalRouter'
 import { MainLayout } from '@renderer/layouts/Main'
 
+import { PortfolioSidebarLink } from './PortfolioSidebarLink'
+
 export const PortfolioPage = () => {
   const { t } = useTranslation('pages', { keyPrefix: 'portfolio' })
-  const { accounts } = useAccountsSelector()
-  const { wallets } = useWalletsSelector()
-  const balanceExchange = useBalancesAndExchange(accounts)
+  const { modalNavigateWrapper } = useModalNavigate()
 
   return (
     <MainLayout
       heading={t('title')}
-      rightComponent={<IconButton icon={<TbRefresh />} size="md" text={t('buttonRefreshLabel')} disabled />}
-      contentClassName="gap-y-3"
+      rightComponent={
+        <div className="flex gap-x-2">
+          <IconButton icon={<MdAdd />} size="md" text={t('newWalletButtonLabel')} disabled />
+          <IconButton
+            icon={<TbFileImport />}
+            size="md"
+            text={t('importButtonLabel')}
+            onClick={modalNavigateWrapper('import')}
+          />
+          <IconButton icon={<TbFileExport />} size="md" text={t('exportButtonLabel')} disabled />
+        </div>
+      }
+      contentClassName="flex-row gap-x-3"
     >
-      <BalanceChartPanel balanceExchange={balanceExchange} accounts={accounts} wallets={wallets} />
-      <BlockchainDataPanel balanceExchange={balanceExchange} accounts={accounts} />
+      <section className="bg-gray-800 rounded drop-shadow-lg max-w-[11.625rem] min-w-[11.625rem] w-full flex flex-col">
+        <div className="text-sm px-4">
+          <p className="py-3">{t('allAccounts')}</p>
+          <Separator />
+        </div>
+        <ul className="max-w-full w-full">
+          <PortfolioSidebarLink title={t('overview')} to="/portfolio/overview" />
+          <PortfolioSidebarLink title={t('allActivity')} to="/portfolio/activity" />
+          <PortfolioSidebarLink
+            title={t('allConnections')}
+            type="button"
+            onClick={modalNavigateWrapper('dapp-connection-list')}
+          />
```

Comment:
You should verify if WalletConnect is initialized to let the user clicks here.
```suggestion
import { EStatus } from '@cityofzion/wallet-connect-sdk-wallet-core'
import { useWalletConnectWallet } from '@cityofzion/wallet-connect-sdk-wallet-react'

  const { status } = useWalletConnectWallet()
  
          <PortfolioSidebarLink
            title={t('allConnections')}
            type="button"
            disabled={status !== EStatus.STARTED}
            onClick={modalNavigateWrapper('dapp-connection-list')}
          />
```